src-gui/src/renderer/components/pages/swap/swap/init/walletAndMakers/WalletAndMakers.tsx

```diff
@@ -0,0 +1,184 @@
+          }}
+        >
+          <Box sx={{ flexGrow: 1, flexShrink: 0, minWidth: "12em" }}>
+            <Typography variant="body1">Bitcoin Balance</Typography>
+            <Typography variant="h5">
+              <SatsAmount amount={max_giveable} />
+            </Typography>
+          </Box>
+
+          <Divider
+            orientation="vertical"
+            flexItem
+            sx={{
+              marginX: { xs: 0, md: 1 },
+              marginY: { xs: 1, md: 0 },
+              display: { xs: "none", md: "block" },
+            }}
+          />
+          <Divider
+            orientation="horizontal"
+            flexItem
+            sx={{
+              marginX: { xs: 0, md: 1 },
+              marginY: { xs: 1, md: 0 },
+              display: { xs: "block", md: "none" },
+            }}
+          />
+
+          <Box
+            sx={{
+              flexShrink: 1,
+              display: "flex",
+              flexDirection: { xs: "row", md: "column", lg: "row" },
+              gap: 2,
+            }}
+          >
+            <Box
+              sx={{
+                display: "flex",
+                flexDirection: "column",
+                gap: 1,
+              }}
+            >
+              <Typography variant="body1">Deposit</Typography>
+              <Typography variant="body2" color="text.secondary">
+                Send Bitcoin to your internal wallet to swap your desired amount
+                of Monero
+              </Typography>
+              <ActionableMonospaceTextBox content={deposit_address} />
+            </Box>
+            <Box
+              sx={{
+                display: "flex",
+                justifyContent: "center",
+                width: "100%",
+                maxWidth: "8em",
+              }}
+            >
+              <BitcoinQrCode address={deposit_address} />
+            </Box>
+          </Box>
+        </Paper>
+
+        {/* Available Makers Section */}
+        <Box>
+          <Box
+            sx={{
+              display: "flex",
+              alignItems: "center",
+              justifyContent: "space-between",
+              mb: 2,
+            }}
+          >
+            <Typography variant="h5">Select an offer</Typography>
+          </Box>
+
+          {/* Maker Discovery Status */}
+          <MakerDiscoveryStatus />
+
+          {/* Real Maker Offers */}
+          <Box>
+            {pendingSelectMakerApprovals.length > 0 && (
+              <Box sx={{ display: "flex", flexDirection: "column", gap: 1 }}>
+                {makerOffers.map((quote, index) => {
+                  return (
+                    <MakerOfferItem
+                      key={index}
+                      quoteWithAddress={quote.quoteWithAddress}
+                      requestId={quote.requestId}
+                    />
+                  );
+                })}
+              </Box>
+            )}
+
+            {/* TODO: Differentiate between no makers found and still loading */}
```

Comment:
Do this. This'll require some changes to the Rust. Maybe @Einliterflasche can help here. If it's too complicated, don't do it and do it in another PR later on.